snap/info.go

```diff
@@ -255,6 +256,11 @@ type ChannelSnapInfo struct {
 
 // Name returns the blessed name for the snap.
 func (s *Info) Name() string {
```

Comment:
SnapSetup.Name has very similar logic, so it probably needs to be changed also at the same time.

We have roughly 300+ places using these, honestly wondering if keeping Name() instead of renaming it to LocalName, is a recipe for short PRs or subtle bugs. It's a bit hard to tell from this PR alone.

Reply:
I think I agree as well. Keeping the function "Name" do to what it used to and introducing additional functions is a safer way to perform a transition.

Reply:
my point btw was not to keep it, but to have all new names forcing to consider each of those 300+ places in turn (once things are in order from that world we could go back to rename one of them to just Name if it's common, makes sense)